CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-# main [(unreleased)](https://github.com/fastruby/next_rails/compare/v1.1.0...main)
+# main [(unreleased)](https://github.com/fastruby/next_rails/compare/v1.2.0...main)
+
+# v1.2.0 / 2022-08-12 [(commits)](https://github.com/fastruby/next_rails/compare/v1.1.0...v1.2.0)
```

Comment:
Looks great, one small thing related to your PR, can we align line 5?
It looks like it has an extra space in front.

Reply:
hey @fbuys I looked it up on my editor and here as well https://github.com/fastruby/next_rails/blob/f534dff96eea533e41ec7c00dfaa0f2009dad12b/CHANGELOG.md

It looks aligned to me. 

I don't know why it looks off on the diff though.